docs/tour/scale-and-distribute.mdx

```diff
@@ -9,17 +9,127 @@ import washboard_hello from '../images/washboard_hello.png';
+
+```txt
+2024-10-20T19:29:30.897232Z  INFO log: wasmcloud_host::wasmbus::handler: Greeting Bob component_id="rust_hello_world-http_component" level=Level::Info context=""
+2024-10-20T19:29:30.897253Z  INFO log: wasmcloud_host::wasmbus::handler: Sleep for 2 to simulate longer processing time component_id="rust_hello_world-http_component" level=Level::Info context=""
+2024-10-20T19:29:32.905355Z  INFO log: wasmcloud_host::wasmbus::handler: Replying greeting 'Hello x1, Bob!' component_id="rust_hello_world-http_component" level=Level::Info context=""
+2024-10-20T19:29:32.906138Z  INFO log: wasmcloud_host::wasmbus::handler: Greeting Bob component_id="rust_hello_world-http_component" level=Level::Info context=""
+2024-10-20T19:29:32.906258Z  INFO log: wasmcloud_host::wasmbus::handler: Sleep for 2 to simulate longer processing time component_id="rust_hello_world-http_component" level=Level::Info context=""
+2024-10-20T19:29:34.914152Z  INFO log: wasmcloud_host::wasmbus::handler: Replying greeting 'Hello x2, Bob!' component_id="rust_hello_world-http_component" level=Level::Info context=""
+2024-10-20T19:29:34.914992Z  INFO log: wasmcloud_host::wasmbus::handler: Greeting Bob component_id="rust_hello_world-http_component" level=Level::Info context=""
+2024-10-20T19:29:34.915023Z  INFO log: wasmcloud_host::wasmbus::handler: Sleep for 2 to simulate longer processing time component_id="rust_hello_world-http_component" level=Level::Info context=""
+2024-10-20T19:29:36.923568Z  INFO log: wasmcloud_host::wasmbus::handler: Replying greeting 'Hello x3, Bob!' component_id="rust_hello_world-http_component" level=Level::Info context=""
+2024-10-20T19:29:36.924326Z  INFO log: wasmcloud_host::wasmbus::handler: Greeting Bob component_id="rust_hello_world-http_component" level=Level::Info context=""
+2024-10-20T19:29:36.924351Z  INFO log: wasmcloud_host::wasmbus::handler: Sleep for 2 to simulate longer processing time component_id="rust_hello_world-http_component" level=Level::Info context=""
+2024-10-20T19:29:38.933227Z  INFO log: wasmcloud_host::wasmbus::handler: Replying greeting 'Hello x4, Bob!' component_id="rust_hello_world-http_component" level=Level::Info context=""
+```
 
-Let's scale up our hello world application to 100 replicas by editing `wadm.yaml`:
+:::note[Checking the `DEBUG` logs of the wasmCloud host]
+You can also check the wasmCloud host's `DEBUG` logs for more detailed information. In these logs, you can clearly see that for received requests, our hello component is instantiated sequentially.
+:::
+
+If you wish, you can also use `wash spy` to check which messages the capability providers and the hello component have received and sent
+
+```bash
+wash spy --experimental rust_hello_world-http_component
+```
+
+:::note[Why are not all requests forwarded to our component?]
+TBD
```

Comment:
I have tried to find out why only 4 requests are forwarded to the Hello component instead of all 10. Or why no more requests are generally forwarded to the component afterwards. Unfortunately, I have not yet found an answer. If I send further requests with `curl`, they are received by the http capability provider and forwarded to the component, but the messages are not received by the component because no log message is logged by it.

Output of wasmCloud logs for the curl request:

```txt
2024-10-26T21:35:08.420377Z DEBUG handle_request{target="rust_hello_world-http_component" scheme="http" authority="localhost:8080" request=Request { method: GET, uri: /?name=Alice, version: HTTP/1.1, headers: {"host": "localhost:8080", "user-agent": "curl/8.7.1", "accept": "*/*"}, body: Body(UnsyncBoxBody) }}: wrpc_transport::invoke: invoking function
```

The expected log messages about scaling the component are missing. Does anybody have a pointer how to debug this?

Reply:
The low level reason for this is still not clear for me. However, I tried to provide a high-level "explanation". Is this good enough for now?

Should I create a bug issue in the wasmCloud repo?